metashare/api/filters.py

```diff
@@ -9,3 +9,9 @@ class ProductFilter(filters.FilterSet):
     class Meta:
         model = Product
         fields = ("slug",)
+
+
+class ProjectFilter(filters.FilterSet):
+    class Meta:
+        model = Project
+        fields = ("product",)
```

Comment:
If the `?product=` query string is not a valid product, this produces an unexpected `400` error:

```
{"product":["Select a valid choice. That choice is not one of the available choices."]}
```

I guess that shouldn't really happen in practice, but it does strike me as surprising.